JASP-Engine/JASP/R/regressionloglinearbayesian.R

```diff
@@ -660,32 +826,41 @@ RegressionLogLinearBayesian <- function(dataset, options, perform="run", callbac
 	
 ########################################################################	
 	if (perform == "init") {
-
-		list(results=results, status="inited")
-		
+	    if (options$counts == "" && numberOfModels < 2) {
+	        endResults <- list(results=results, status="complete") #, keep=keep)
+	        return(endResults)
+	    } else if (options$counts != "" && numberOfModels < 1) {
+	        endResults <- list(results=results, status="complete") #, keep=keep)
+	        return(endResults)
+	    } else {
+	        endResults <- list(results=results, status="inited") #, state=state, keep=keep)
+	        return(endResults)
+	    }
 	} else {
-	
-		list(results=results, status="complete")
+	    endResults <- list(results=results, status="complete")
+	    return(endResults)
```

Comment:
If you insist on making the temporary "endResults" you could place a single "return(endResults)" after the if statement. Also, you do the same thing in all of the if statements except that status = "inited" in one of them. Both are not super relevant but would result in slightly cleaner code.